css/slider.css

```diff
@@ -0,0 +1,33 @@
+.slider-container {
+    width: 30rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    gap: var(--size-1-5);
+}
+
+
+input.slider {
+    -webkit-appearance: none;
+    background-color: var(--secondary-color);
+    position: relative;
+    height: 30px;
```

Comment:
In height use you have use 30px ..but we should use rem for more than 10px .. vikram sir  also said in his session.. rest  All looks perfect ..

Reply:
Done! Thanks